cli.js

```diff
@@ -37,9 +37,8 @@ const disableHooksParser = v => disabledHooks.push(v);
 const showError = err => {
   console.error(red('Something went wrong:'));
   console.error(red(err.stack || err.message));
-  if (err.errors) {
-    console.error(red(JSON.stringify(err.errors)));
-  }
+  if (err.errors) console.error(red(JSON.stringify(err.errors)));
+  if (err.validationErrors) console.error(red(JSON.stringify(err.validationErrors, null, 4)));
```

Comment:
@fmvilas any objections to this one? when working on parser I noticed generator is not giving all needed info in case of validation issue

Reply:
No objections. I think it's a great addition 👍